src/js/app.js

```diff
@@ -21,29 +21,33 @@ const app = new Vue({
     radical_selection: undefined,
 
     searching: false,
+
+    isDropdownOpen: false,
+    isIncludeSimilar: false,
```

Comment:
~~Can we not use the `include_similar` data property?~~ (We can rename it to `is_include_similar`... or maybe `should_include_similar`?... If y'want.)

EDIT: actually, let's just remove this (see the comments below about `v-model`).

Reply:
(Also, sorry I didn't mention it anywhere... I'm using `snake_case` for js variables / properties / keys / etc and `camelCase` for method names.)